faq.rst

```diff
@@ -221,3 +221,7 @@ you can see the pvc/prometheus-data is the persistent volume claim that is mount
 
   oc delete all -l app=busybox
 
+How long to we keep application logs?
+-------------------------------------
+
+We keep application logs for 10 days.
```

Comment:
I think this should be explained a bit more... F.e. that this applies to logs stored in elasticsearch, accessible via Kibana. And I would write about the word "retention" as this is more descriptive. TL;DR: More prose needed.